draft-ietf-httpbis-http2.xml

```diff
@@ -773,34 +773,34 @@ HTTP2-Settings    = token68
         <figure anchor="StreamStatesFigure" title="Stream States">
           <artwork type="drawing">
             <![CDATA[
-                             +--------+
-                     send PP |        | recv PP
+                     send PP +--------+ recv PP
                     ,--------|  idle  |--------.
-                   /         |        |         \
-                  v          +--------+          v
+                   v         +--------+         v
            +----------+          |           +----------+
-           |          |          | send H/   |          |
-     ,-----| reserved |          | recv H    | reserved |-----.
-     |     | (local)  |          |           | (remote) |     |
-     |     +----------+          v           +----------+     |
-     |         |             +--------+             |         |
-     |         |     recv ES |        | send ES     |         |
-     |  send H |     ,-------|  open  |-------.     | recv H  |
-     |         |    /        |        |        \    |         |
-     |         v   v         +--------+         v   v         |
-     |     +----------+          |           +----------+     |
-     |     |   half   |          |           |   half   |     |
-     |     |  closed  |          | send R/   |  closed  |     |
-     |     | (remote) |          | recv R    | (local)  |     |
+     ,-----| reserved |          | send H/   | reserved |-----.
+     |     | (local)  |          | recv H    | (remote) |     |
      |     +----------+          |           +----------+     |
-     |          |                |                 |          |
-     |          | send ES/       |        recv ES/ |          |
-     |          | send R/        v         send R/ |          |
-     |          | recv R     +--------+    recv R  |          |
-     | send R/  `----------->|        |<-----------'  send R/ |
-     | recv R                | closed |               recv R  |
-     `---------------------->|        |<----------------------'
-                             +--------+
+     |         |                 v                  |         |
+     |         |     recv ES +--------+ send ES     |         |
+     |  send H |     ,-------|  open  |-------.     | recv H  |
+     |         |    /        +--------+        \    |         |
+     |         v   v          /  |              v   v         |
+     |     +----------+      /   |           +----------+     |
+     |     |   half   |     /    |           |   half   |     |
+     |     |  closed  |    /     | send R    |  closed  |     |
+     |     | (remote) |   /      |           | (local)  |     |
+     |     +----------+  .       |           +----------+     |

```

Comment:
nit: `,` seems to be the angled character in this direction...did this not look pretty here?